pkg/clients/edgeconnect/client_test.go

```diff
@@ -188,7 +188,7 @@ func edgeConnectGetServerHandler() http.HandlerFunc {
 			writeOauthTokenResponse(writer)
 		case fmt.Sprintf("/edge-connects/%s", EdgeConnectID):
 			writer.WriteHeader(http.StatusOK)
-			resp := GetResponse{
+			resp := EdgeConnectResponse{
```

Comment:
If we follow @andriisoldatenko pattern I think GetResponse would be better. What do you think?